source/common/runtime/runtime_impl.cc

```diff
@@ -38,6 +42,21 @@ void countDeprecatedFeatureUseInternal(const RuntimeStats& stats) {
   stats.deprecated_feature_seen_since_process_start_.inc();
 }
 
+#ifdef ENVOY_ENABLE_QUIC
+void refreshQuicReloadableFlags(const Snapshot::EntryMap& flag_map) {
+  for (const auto& it : flag_map) {
+    if (absl::StartsWith(it.first, quiche::EnvoyQuicheFlagPrefix) &&
+        it.second.bool_value_.has_value()) {
+      auto* res = quiche::FlagRegistry::getInstance().findFlag(it.first);
```

Comment:
Consider adding a separate accessor that can be used to get the corresponding QUICHE flag based on runtime feature name.

Reply:
done.